packages/core/styles/common/variables.pcss

```
@@ -149,4 +149,10 @@
   --ifm-z-index-overlay: 400;
 }
 
+@media (prefers-reduced-motion: reduce) {
+  :root {
+    --ifm-transition-fast: 0ms;
+  }
+}
```

```suggestion
@media (prefers-reduced-motion: reduce) {
  :root {
    --ifm-transition-fast: 0ms;
    --ifm-transition-slow: 0ms;
  }
}
```

Just removing the slow transitions too, although I'm not sure we actually use any 😅  maybe we will someday